src/views/QuantumTP6/index.js

```diff
@@ -2,10 +2,10 @@ import React from 'react';
 import { withStyles } from '@material-ui/core';
 import PropTypes from 'prop-types';
 import Grid from '@material-ui/core/Grid';
-import { frum } from '../utils/query_ops';
-import { TP6_PAGES } from './config';
-import DashboardPage from '../components/DashboardPage/index';
-import PerfherderGraphContainer from '../containers/PerfherderGraphContainer/index';
+import frum from '../../utils/queryOps';
+import { TP6_PAGES } from '../../quantum/config';
+import DashboardPage from '../../components/DashboardPage/index';
```

Comment:
You need to add `index` at the end of the path. It's implicit.
```suggestion
import DashboardPage from '../../components/DashboardPage';
```